From fc13c47a3cc4b9b982edc13f84fd59382de1a880 Mon Sep 17 00:00:00 2001
From: Daniel Bradley <daniel@pulumi.com>
Date: Thu, 17 Nov 2022 09:55:46 +0000
Subject: [PATCH 15/28] Un-deprecate RDS instance name

We don't have a good migration story yet so this just adds noise for users.

Reference: https://github.com/pulumi/pulumi-aws/issues/2203
---
 internal/service/rds/instance.go | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/internal/service/rds/instance.go b/internal/service/rds/instance.go
index bb931a02b3..12f5311916 100644
--- a/internal/service/rds/instance.go
+++ b/internal/service/rds/instance.go
@@ -400,6 +400,16 @@ func ResourceInstance() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+				ConflictsWith: []string{
+					"db_name",
+					"replicate_source_db",
+				},
+			},
 			"nchar_character_set_name": {
 				Type:     schema.TypeString,
 				Optional: true,
